src/content/docs/reference/cli.mdx

```diff
@@ -4,9 +4,23 @@ description: A reference of supported operations using the Daytona CLI.
 sidebar:
   label: Daytona CLI Reference
 ---
+import Aside from "@components/Aside.astro";
 
-The `daytona` CLI provides access to all of Daytona's core features.
-You can access help for each command using the `--help`/`-h` flag.
+The `daytona` command-line tool provides access to Daytona's core features.
+The tool acts as both an interface to a running Daytona Server, and as an interface to start and stop the server.
+
+You can use the `daytona` tool for the following operations:
+
+*   Managing the lifecycle of the Daytona Server.
+*   Managing [Workspaces](../usage/workspaces), [Git Providers](../configuration/git-providers), [Providers](../configuration/providers), and other Daytona components.
+*   Configuring the [Daytona Server](./server) interactively.
+
+This reference lists all commands supported by the `daytona` command-line tool complete with a description of their behaviour, and any supported flags.
+You can access this documentation on a per-command basis by appending the `--help`/`-h` flag when invoking `daytona`.
+
+<Aside type="note">
+This reference does not apply to the `daytona` command when run inside of a Workspace.
```

Comment:
@osslate why this note? Daytona works in the workspaces also, albeit there are some differences. We should maybe just flag those commands that are only outside of workspace? cc @idagelic as I don't know which those are.

Reply:
A separate set of CLI docs are generated for the Daytona command while run in a Workspace at the moment, but they're not handled by the script currently. The note is to clarify that to users while we figure out the best course of action to include them in the docs.

I believe there are a few ways in which some of the subcommands differ in syntax between the two contexts (host vs. in a Workspace), so simply marking commands that exist in both may not work.

Reply:
Answered in Slack

Reply:
We've settled on splitting this out into two references. Let's ignore the "Workspace CLI" reference for this PR as it'll be a new page + require updating the generation script.

Tracking those changes in https://github.com/daytonaio/docs/issues/65